### PR TITLE
Added various tests

### PR DESCRIPTION
**Pull request by radarhere**
Contributing to #722

### CODE REVIEW

File: Tests/test_file_gbr.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from helper import unittest, PillowTestCase
+
+from PIL import Image, GbrImagePlugin
```
Comment (hugovk) [inline Tests/test_file_gbr.py]: 'Image' imported but unused in a lot of these

File: Tests/test_file_gbr.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class TestFileGbr(PillowTestCase):
+
+    def test_invalid_file(self):
+        self.assertRaises(SyntaxError, lambda: GbrImagePlugin.GbrImageFile("Tests/images/flower.jpg"))
```
Comment (hugovk) [inline Tests/test_file_gbr.py]: Not so important, but > 80

**Comment by hugovk** [comment]
When merged, we should notify/PR upstream the olefile test changes.

File: Tests/test_file_gbr.py (hunk 1 above)
Comment (radarhere) [inline Tests/test_file_gbr.py]: Yep, fixed, thanks

File: Tests/test_file_gbr.py (hunk 2 above)
Comment (radarhere) [inline Tests/test_file_gbr.py]: I tried breaking between SyntaxError and lambda, or between lambda and GbrImagePlugin, but it still leaves a line > 80

Comment (hugovk) [inline Tests/test_file_gbr.py]: This works:
```python
    def test_invalid_file(self):
        self.assertRaises(
            SyntaxError,
            lambda: GbrImagePlugin.GbrImageFile("Tests/images/flower.jpg"))

```

Comment (radarhere) [inline Tests/test_file_gbr.py]: Depending on where the indent is, that can cause flake8 under-or-over-intended warnings. I've added another commit to fix flake8 warnings from the first commit, and a few other Tests as well.

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**